components/dialog/dialog-fullscreen.js

```diff
@@ -196,7 +205,8 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 	}
 
 	render() {
-
+		if (!this.width) this.width = 1170;
```

Comment:
It looks like this would cause another iteration of the Lit life cycle. Can we just set the default in the constructor to 1170 instead of 0?